eduNEXT/eox-audit-model
# chore(ci): remove integration tests workflow

### Description

This PR removes the existing integration test and the workflow for integration testing.

#### Reason:
This plugin is a dependency that neither uses nor imports any definitions from the Open edX platform. Additionally, the existing integration test is no longer relevant since integration tests are now run in a completely isolated environment, separate from the platform (for more information, you can visit  https://github.com/eduNEXT/integration-test-in-tutor).

### How to Test:
Verify that the integration test check is no longer executed in the CI/CD pipeline of the PR.
